governance/reviews/template.md

```diff
@@ -62,28 +87,92 @@ Over the next year, the project should work on the following issues to improve i
 
 <!--- Provide the commit of the file under evaluation as a point-in-time reference to this review. --->
 
-The following table details the governance areas expected for a project. Coverage is indicated by Complete, Partial, and Missing.
+The following table details the governance areas expected for a project. Coverage is indicated by Complete, Partial, Missing, and Unknown.
 * Complete - the content of the governance documentation is fully detailed and does not leave any question to the reader.
 * Partial - the content of the governance documentation is missing some information and would leave the reader with questions or some level of misunderstanding.
 * Missing - the documentation is absent, wholly undiscoverable, or woefully inadequate in meeting the objectives of that governance content. The reader cannot act on the content that is available.
 * Unknown - status cannot be assessed at this time
 
-| Governance Area | Coverage | Documents | Finding Notes |
-|:----------------|:--------:|:------:|:--------------|
-| Project Purpose | Complete/Partial/Missing | *LINKS* | |
-| Maintainer List | Complete/Partial/Missing | *LINKS* | |
-| Code of Conduct | Complete/Partial/Missing | *LINKS* | |
-| Contributor Guide | Complete/Partial/Missing | *LINKS* | |
-| Contributor Ladder | Complete/Partial/Missing | *LINKS* | |
-| Maintainer Lifecycle | Complete/Partial/Missing | *LINKS* | |
-| Decision-making | Complete/Partial/Missing | *LINKS* | |
-| Code and Docs Ownership | Complete/Partial/Missing | *LINKS* | |
-| Security Reporting and response | Complete/Partial/Missing | *LINKS* | |
-| Communication and Meetings | Complete/Partial/Missing | *LINKS* | |
+<!--- 
+Provide the links in the table to the documents/pages that are relevant to the governance area. The links should be to the permanent addresses to snapshots (commits) of the file under evaluation as a point-in-time reference to this review. To ensure consistency, use same commits as links for multiple files within a single repository.
+--->
+
+<!--- 
```

Comment:
Starting here, this is feeling like this needs to go into an actual Guide To Reviewers doc, instead of inline comments.

Reply:
I am ok removing these detailed instructions, if you think a future guide is a better place.

I wouldn't keep the instructions in 2 places as they will become out-of-sync eventually.

Reply:
I removed the detailed comments now.